### DISCUSSION TITLE
How to use webcam on V2 ??

### DISCUSSION THREAD

**Discussion by akshays-repo** [wailsapp/wails#1935] [Q&A]
I'm using React for the frontend and i can't access webcam,

**Comment by alanchenchen** [comment]
Pure webRTC needs https protocal.
![image](https://user-images.githubusercontent.com/23295628/199152193-9d8c13e6-2528-4e28-a9c5-a37f4cc1aee8.png)

You should use  another native go module to access webcam

**Reply by alanchenchen** [reply]
https://github.com/wailsapp/wails/discussions/883 will be helpful for you
